glide.lock

```diff
@@ -169,4 +184,63 @@ imports:
   - pkg/util/yaml
   - pkg/watch
   - pkg/watch/versioned
+- name: k8s.io/kubernetes
```

Comment:
@kadel I agree on the size part, but I'd really want this to happen, since this adds so much value and itegrity to the generated artifacts, in the sense that they will not fail to get created on the cluster.
Is there some other way we can do this? Maybe by not making this a part of the code, or running these only in tests, or some way?